packages/amplify-graphql-transformer-core/src/types/import-appsync-api-types.ts

```diff
@@ -29,5 +29,6 @@ export type RDSConnectionSecrets = TransformerSecrets & {
 
 export const MYSQL_DB_TYPE = 'MySQL';
 export const DDB_DB_TYPE = 'DDB';
+export const POSTGRES_DB_TYPE = 'Postgres';
 
-export type ModelDatasourceType = 'MySQL' | 'DDB';
+export type ModelDatasourceType = 'MySQL' | 'DDB' | 'Postgres';
```

Comment:
nit: Please alphabetize. This has the added effect of putting the most commonly used/default option (DDB) first
```suggestion
export type ModelDatasourceType = 'DDB' | 'MySQL' | 'Postgres';
```